styles.css

```diff
@@ -419,3 +420,322 @@ form {
   text-align: center;
   margin-left: 125px;
 }
+
+@media (min-width: 768px) {
+  /* .............header section.......... */
```

Comment:
- [ ] I noticed that your design isn't responsive. On larger desktop screens, it is well positioned but on other desktop screens, there are white patches to the right of the design from the Headline to the Footer. Kindly adjust your design to be responsive. I would advise you to use the CSS `grid` layout for the Works section for the desktop version. 👍 

![chrome-capture-2023-1-23](https://user-images.githubusercontent.com/77106962/220784235-9555e50e-0aa9-4727-8cf3-ee2882fbd18a.gif)